### PR TITLE
Fixes #14: Removes old implementations and updates documentaion

### LINKED ISSUE #14: Publish library in Maven Repository
Publish loklak_jlib_api in MavenCentral repository.

TODOs for publishing the library:
- [x] Making loklak_jlib_api similar to other Loklak API libraries, by removing the scraping(tweet-harvesting) code. The scraping part in the library is implemented by classes in `harvester` and `objects` packages.
- [x] The `client` package have API endpoints implementation using Java reflect and proxy (implemented by `LoklakAPI.java` and `APIGenerator.java`) and the old API implementations i.e. `PushClient`, `SearchClient` and `SuggestClient` can be removed.
- [x] Removing ant build as Maven has already been added. Including a new library (which has dependencies) to provide ant support will increase a lot of complexity (need to manually download jar files and provide it in `lib` directory).
- [ ] Add a markdown file, that gives examples to use Loklak API endpoints.

### PR DESCRIPTION
**Pull request by SKrPl**
Fixes issue: https://github.com/loklak/loklak_jlib_api/issues/14

Removes the following things:
* Old API endpoint implementations and scraping code along with their unit-tests.
* Deprecated ant "build.xml" and ".jar" files in "lib" directory.
* Eclipse project files ".project" and ".classpath".

Modifies the README.md file.

### CODE REVIEW

**Comment by SKrPl** [comment]
@mariobehling @Orbiter @sudheesh001 @daminisatya @Achint08 @djmgit @hemantjadon @kavithaenair @singhpratyush @vibhcool Please review.

**Review by vibhcool — APPROVED** [review]
Looks good to me :)

**Review by singhpratyush — APPROVED** [review]

**Review by kavithaenair — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,35 +1,67 @@
...
+```java
+public class LoklakApiUse {
+    
+    public static void main(String[] argv) {
+        String baseUrl = "https://api.loklak.org";
+        // API methods are generated by the library.
+        LoklakAPI loklakApi = APIGenerator.createApiMethods(LoklakAPI.class, baseUrl);
+        
+        // using the hello API endpoint, request url: http://api.loklak.org/api/hello.json
+        JSONOBject helloReply = loklakApi.hello(); // use the required API endpoint with parameters
```
Comment (sudheesh001) [inline README.md:59]: `JSONObject`, there's a typo error here.

**Review by sudheesh001 — COMMENTED** [review]
Where are the removed `*Client.java` files moved if they're not anymore in this repository. Did you add them correctly to the relevant susi and loklak repositories?

@Orbiter Can take a final look. I just left some feedback.

**Comment by SKrPl** [comment]
@sudheesh001 I couldn't find any `loklak_jlib.jar` file being used in loklak_server and susi_server. So, I think they do not depend on this library.

File: README.md (hunk 1 above)
Comment (SKrPl) [reply README.md:59]: Done.

**Review by Orbiter — APPROVED** [review]

**Review by sudheesh001 — APPROVED** [review]
Looks good. We can merge this.

**Pull request merged**